app/models/non_hmis_client.rb

```diff
@@ -1,7 +1,7 @@
 class NonHmisClient < ActiveRecord::Base
   has_one :project_client, ->  do
     where(
-        data_source_id: DataSource.where(db_identifier: 'Deidentified').select(:id),
+        data_source_id: NonHmisClient.data_source_id
```

Comment:
This change forces an extra SQL query that is unnecessary.

Reply:
Putting it back to the way it was.